src/hooks/useMediaDevices.ts

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useState } from "react";
+
+type MediaDevices = {
+    cameraList: InputDeviceInfo[];
+    microphoneList: InputDeviceInfo[];
+    setCameraHandler: (device: InputDeviceInfo) => void;
```

Comment:
please use name `selectCamera` and `selectMicrophone`

Reply:
I'm a bit worried about using this name because the hook will also export some values named `selectedCamera` and `selectedMicrophone` which look very similar to your names.